site/en/release/index.md

```diff
@@ -117,6 +117,7 @@ For LTS releases, the procedure and policies below are followed:
    "`@bazel-io flag`" on relevant GitHub issues or PRs to mark them as potential
         release blockers, the Bazel team triages them and decide whether to
         back-port the commits.
+    *   A user may also choose to cherry-pick certain commit(s) on their own by creating a [cherry-pick request](https://github.com/bazelbuild/bazel/issues/new?assignees=&labels=&projects=&template=cherry_pick_request.yml){: .external}. Fill in the Title, Commit ID(s), Category, and Reviewer(s). If there are multiple commits or reviewers, then separate them by commas. After, set the milestone of the issue by clicking on the milestone setting and choosing the proper X.Y.Z release blockers (this will trigger the cherry-pick bot to cherry-pick the commits to the "release-X.Y.Z" branch). Finally, submit the issue. The cherry-pick bot will notify whether or not the commits have been successfully cherry-picked in the new issue. See [example](https://github.com/bazelbuild/bazel/issues/20230){: .external}.
```

Comment:
"A user may also choose to cherry-pick certain commits" is not accurate. First, not all requests will be accepted, it depends on the author and reviewer of the commits. Also, a public user cannot add milestone to the issue, so this is limited to Bazel maintainers to use this issue.